app/presenters/order_detail_notice_presenter.rb

```diff
@@ -0,0 +1,85 @@
+class OrderDetailNoticePresenter < DelegateClass(OrderDetail)
+
```

Comment:
Is there a functional reason why `DelegateClass` is better than `SimpleDelegator`?